package.json

```diff
@@ -65,81 +94,131 @@
       "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
     }
   },
-  "scripts": {
-    "bootstrap:hoist": "lerna bootstrap --hoist",
-    "clean": "lerna clean --yes && rm -rf node_modules || true",
-    "clean:install": "npm run clean && npm install",
-    "clean:obsolete-snapshots": "npm test -- -u",
-    "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
-    "compile:clean": "rm -rf packages/**/lib",
-    "compile:src": "lerna run compile",
-    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p",
-    "compile:lowlight": "rm -rf ./build; THEME=clinical-lowlight-theme webpack -p",
-    "compile:fusion": "rm -rf ./build; THEME=orion-fusion-theme webpack -p",
-    "danger": "danger ci",
-    "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
-    "docs": "node scripts/root-readme-updater/index.js",
-    "docs:peerDependencies": "node scripts/peer-dependency-generator/index.js",
-    "heroku-prebuild": "npx lerna init",
-    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
-    "jest": "jest",
-    "jest:coverage": "jest --coverage",
-    "link-parent-bin": "link-parent-bin",
-    "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --ext .js,.jsx .",
-    "lint:scss": "stylelint 'packages/**/src/**/*.scss' --syntax scss",
-    "pretest": "npm run lint",
-    "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
-    "start": "tt-serve",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
-    "test": "npm run lint && npm run jest && npm run wdio",
-    "wdio-default": "wdio",
-    "wdio-lowlight": "NODE_OPTIONS=--max-old-space-size=2048 THEME=clinical-lowlight-theme wdio",
-    "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+  "dependencies": {
+    "d3": "^5.16.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.5.0",
     "@babel/core": "^7.5.0",
     "@babel/plugin-proposal-object-rest-spread": "^7.5.0",
     "@babel/plugin-transform-object-assign": "^7.2.0",
     "@babel/plugin-transform-runtime": "^7.5.0",
-    "@babel/preset-env": "^7.5.0",
+    "@babel/polyfill": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.0.0",
-    "@cerner/eslint-config-terra": "^4.0.0",
+    "@babel/runtime": "^7.10.4",
     "@cerner/browserslist-config-terra": "^3.0.0",
+    "@cerner/carbon-site-helpers": "^1.11.3",
```

Comment:
What is in here that we need in the mono repo?

If we still need it we should add it to the mono repo as another package.

Reply:
This repo has the logic for rendering the current carbon dev site. From my understanding, this dependency will be replaced by the terra-dev-site implementation in the future.